### PR TITLE
feat: Java :implementation of classic load balancer sample

### PR DESCRIPTION
**Pull request by djiten75**
Fixes # 

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

**Comment by rhboyd** [comment]
You can add commits to a PR. Closing them and opening a new one for each change loses context.

**Comment by NGL321** [comment]
Hey @djiten75! Any update?

File: java/classic-load-balancer/src/main/java/software/amazon/awscdk/examples/ClassicLoadBalancerApp.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package software.amazon.awscdk.examples;
+
+import software.amazon.awscdk.core.App;
+
+public class ClassicLoadBalancerApp{
+    public static void main(final String[] args) {
+        App app = new App();
+
+        new ClassicLoadBalancerStack(app, "cdk-classic-load-balancer-example");
+
```
[the remark this answers is not in the document]
Comment (djiten75) [reply java/classic-load-balancer/src/main/java/software/amazon/awscdk/examples/ClassicLoadBalancerApp.java]: I have checked the other examples where are in master and none of them are camel case and hence I want to be consistent with them. Let me know if we are now drifting from this and if so would the other examples get fixed as well?
Examples are : hello-world, lambda-cron

File: java/classic-load-balancer/pom.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            </plugin>
+
+        </plugins>
+    </build>
+
+    <dependencies>
+        <dependency>
+            <groupId>software.amazon.awscdk</groupId>
+            <artifactId>core</artifactId>
+            <version>[1.17.0.DEVPREVIEW, 2)</version>
```
[the remark this answers is not in the document]
Comment (djiten75) [reply java/classic-load-balancer/pom.xml]: Fixed this.

File: java/classic-load-balancer/src/main/java/software/amazon/awscdk/examples/ClassicLoadBalancerStack.java [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import software.amazon.awscdk.services.ec2.InstanceClass;
+import software.amazon.awscdk.services.ec2.InstanceSize;
+import software.amazon.awscdk.services.elasticloadbalancing.LoadBalancer;
+import software.amazon.awscdk.services.elasticloadbalancing.HealthCheck;
+import software.amazon.awscdk.services.elasticloadbalancing.LoadBalancerListener;
+import software.amazon.awscdk.services.elasticloadbalancing.ListenerPort;
+
+
+/**
+ * Lambda Cron CDK example for Java!
```
[the remark this answers is not in the document]
Comment (djiten75) [reply java/classic-load-balancer/src/main/java/software/amazon/awscdk/examples/ClassicLoadBalancerStack.java]: Fixed

File: java/classic-load-balancer/src/main/java/software/amazon/awscdk/examples/ClassicLoadBalancerStack.java [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+ */
+class ClassicLoadBalancerStack extends Stack {
+    public ClassicLoadBalancerStack(final Construct parent, final String name) {
```
[the remark this answers is not in the document]
Comment (djiten75) [reply java/classic-load-balancer/src/main/java/software/amazon/awscdk/examples/ClassicLoadBalancerStack.java]: Added the constructor

**Comment by djiten75** [comment]
> Hey @djiten75! Any update?

Completed all the review comments

**Pull request merged**